### PR TITLE
Remove chyt-controller and query-tracker from ytsaurus core-like images

### PR DESCRIPTION
**Pull request by achulkov2**
These components are deployed from their own images for quite a while now.

### CODE REVIEW

**Comment by Krock21** [comment]
And it will help us to avoid typo errors when users specify `ytsaurus/ytsaurus` image for queryTracker with a very old version of QT (happened to us)

File: yt/docker/ytsaurus/credits/chyt-controller.CREDITS [hunk 1]
```diff
@@ -1,2007 +0,0 @@
-Go (the standard library)
```
Comment (savnadya) [inline yt/docker/ytsaurus/credits/chyt-controller.CREDITS:1]: It's used in yt/docker/strawberry, let's move it

Comment (achulkov2) [reply yt/docker/ytsaurus/credits/chyt-controller.CREDITS]: Fixed.

**Comment by savnadya** [comment]
Shipped!

**Pull request closed without merging**